server/src/main/java/org/spine3/server/event/EventDelivery.java

```diff
@@ -30,8 +30,8 @@
 import java.util.concurrent.Executor;
 
 /**
- * Base functionality for the routines delivering the {@link org.spine3.base.Event}s to event consumers,
- * such as {@link EventDispatcher}s or {@link EventSubscriber}s.
+ * Base functionality for the routines delivering the {@linkplain org.spine3.base.Event Events}
```

Comment:
@IlliaShepilov This is not a typo, it looks much better now because we can do like this since we use FQNs.